### PR TITLE
Add release script

### PR DESCRIPTION
**Pull request by garrettmoon**
This is my attempt at a script to replace the release process. To use this:
- Make sure you're on clean master
- Set GITHUB_CHANGELOG_API_KEY environment variable
- Run Scripts/release.sh --patch / --minor / --major

### CODE REVIEW

**Comment by garrettmoon** [comment]
It generates a branch like this: https://github.com/pinterest/PINOperation/pull/37

File: Scripts/release.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -e
+
+PROJECT=PINOperation
+
+# BRANCH=$(git rev-parse --abbrev-ref HEAD)
+# if [[ "$BRANCH" != "master" ]]; then
+#     echo "Please run on master branch."
+#     exit 1
+# fi
```
Comment (jparise) [inline Scripts/release.sh:10]: Remove commented lines?

File: Scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        ;;
+    
+    *)
+        echo "Usage release.sh --patch / --minor / --major"
+        exit 1
+        ;;
+esac
+
+PODSPEC="$PROJECT.podspec"
+PODSPEC=`echo $PODSPEC`
```
Comment (jparise) [inline Scripts/release.sh:42]: I'm not sure I understand what this line does.

Comment (garrettmoon) [reply Scripts/release.sh:42]: It used to do something when the above was `PODSPEC="*.podspec"` :)

**Review by jparise — APPROVED** [review]

**Pull request merged**